packages/gatsby-remark-graphviz/__tests__/index.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
```

Comment:
Not sure what's going on here -- but this is a transpiled test? And unrelated to this PR?

Reply:
@leonlafa  could you revert this change? Like @DSchau it's an error :) We fixed this bug ourselves in latest master where __tests__/index.js file was created when running jest